ramls/invoice.raml

```diff
@@ -1,7 +1,7 @@
 #%RAML 1.0
 
 title: Invoice
-version: v3.0
+version: v3.1
```

Comment:
is this version increment necessary? We haven't modified anything in the request/response

Reply:
In scope of MODINVOICE-120 hardcoded voucher "accounting code" was changed to real accounting code from invoice. This affects voucher creation during invoice approval flow, i.e invoice updating. Is this not a reason for minor version increasing?

Reply:
yea that is the reason to increase the interface version . But what does it have to do with the raml version?

Reply:
Yes, correct. I've already reverted. Thanks.